tests/benchmark/stateful/bloatnet/depth_benchmarks/test_worst_case_attack_execute.py

```diff
@@ -0,0 +1,539 @@
+            # Create funding transaction with 1 wei to ensure the account exists in the trie
+            funding_tx = Transaction(
+                to=aux_address,
+                value=1,  # 1 wei to create the account in the trie
+                gas_limit=21_000,
+                sender=deployer_address,
+                max_fee_per_gas=10_000_000_000,  # 10 gwei
+                max_priority_fee_per_gas=1_000_000_000,  # 1 gwei
+            )
+            funding_txs.append(funding_tx)
+
+    # Compile contract for the specified storage depth
+    print(f"  Compiling depth_{storage_depth}.sol...")
+    try:
+        init_code_bytes = compile_worst_case_contract(storage_depth)
+    except FileNotFoundError as e:
+        pytest.skip(f"Skipping test: {e}")
+    # Wrap in Bytecode with proper stack parameters
+    init_code = Bytecode(init_code_bytes, popped_stack_items=0, pushed_stack_items=0)
+
+    # Calculate gas for deployment
+    # CREATE2 gas: 32000 base + 6 * init_code_size + execution
+    init_code_size = len(init_code)
+    # Constructor performs storage_depth SSTOREs to deep slots
+    # Each SSTORE to a new slot costs ~22100 gas (cold storage)
+    constructor_gas = storage_depth * 22100  # Dynamic based on storage depth
+    deployment_gas_per_contract = (
+        32000  # CREATE2 base cost
+        + 6 * init_code_size  # Init code cost (6 gas per byte for CREATE2)
+        + 200 * init_code_size  # Code deposit cost (200 gas per byte)
+        + constructor_gas  # Constructor execution (storage_depth SSTOREs to deep slots)
+    )
+
+    # Calculate how many contracts we can deploy per transaction
+    max_contracts_per_tx = MAX_GAS_PER_TX // deployment_gas_per_contract
+
+    print(f"  Init code size: {init_code_size} bytes")
+    print(f"  Deployment gas per contract: ~{deployment_gas_per_contract:,}")
+    print(f"  Max deployments per tx: {max_contracts_per_tx}")
+
+
+    from ethereum.crypto.hash import keccak256
+
+    # Calculate actual CREATE2 addresses using Nick's deployer
+    actual_contracts = []
+    deployment_txs = []  # Track deployment transactions
+
+    print(f"  Deploying {NUM_CONTRACTS} contracts via Nick's method...")
+    print(f"  Nick's deployer: {NICK_DEPLOYER}")
+
+    for i, contract_data in enumerate(contracts):
+        salt = contract_data["salt"]
+
+        # Convert salt to 32 bytes
+        salt_bytes = salt.to_bytes(32, 'big') if isinstance(salt, int) else bytes.fromhex(salt[2:] if salt.startswith("0x") else salt)
+
+        # Nick's method: calldata = salt (32 bytes) + init_code
+        calldata = salt_bytes + bytes(init_code)
+
+        # Calculate CREATE2 address: keccak256(0xff ++ nick_deployer ++ salt ++ keccak256(init_code))
+        create2_input = (
+            bytes.fromhex("ff") +
+            bytes(NICK_DEPLOYER) +
+            salt_bytes +
+            keccak256(bytes(init_code))
+        )
+        contract_addr = keccak256(create2_input)[-20:]  # Last 20 bytes
```

Comment:
Isn't this part of the json already? `.contract_address` field? Or was that deployed on a different address (so not the `0x4e59b44847b379578588920ca78fbf26c0b4956c`)?